src/utils/firebase.js

```diff
@@ -1,20 +1,21 @@
 import React from 'react';
 import firebase from 'firebase/app';
 import 'firebase/auth';
+import { FirebaseRepository } from 'hackoss';
 
 const FirebaseContext = React.createContext(null);
 
-class FirebaseApp {
-  constructor() {
-    const config = {
-      apiKey: process.env.REACT_APP_FIREBASE_TOKEN || '',
-      authDomain: 'hackoss-admin.firebaseapp.com',
-      databaseURL: 'https://hackoss-admin.firebaseio.com',
-      projectId: 'hackoss-admin',
-      storageBucket: 'hackoss-admin.appspot.com',
-      messagingSenderId: '841474314837',
-    };
-    firebase.initializeApp(config);
+const config = {
```

Comment:
I think everything here should be in the `.env` file, even if they may not be as critical as the API key. Thoughts?

Reply:
Agree, We can do this.